hash/msx1_cass.xml

```diff
@@ -12989,6 +13126,28 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<software name="scpghost">
+		<description>Scapeghost (Europe)</description>
+		<year>1989</year>
+		<publisher>Level 9 Computing</publisher>
+		<info name="usage" value="Load with RUN&quot;CAS:&quot;."/>
+		<part name="cass1" interface="msx_cass">
+			<dataarea name="cass" size="60407">
+				<rom name="scapeghost - level 9 computing (part 1).cas" size="60407" crc="44ec0374" sha1="fd4b912f341d2b9a079e61be4f81ade4e229ac57"/>
+			</dataarea>
+		</part>
+		<part name="cass2" interface="msx_cass">
+			<dataarea name="cass" size="60407">
+				<rom name="scapeghost - level 9 computing (part 2).cas" size="60407" crc="24375acf" sha1="90d08b32b264bd6fcf81794d973d4dbfe0a8f2c5"/>
+			</dataarea>
+		</part>
+		<part name="cass3" interface="msx_cass">
+			<dataarea name="cass" size="60407">
+				<rom name="scapeghost - level 9 computing (part 3).cas" size="60407" crc="f6fdb73d" sha1="58d8c07b8f90c7456ea52610f801e5228830b049"/>
+			</dataarea>
+		</part>
```

Comment:
Do we have labels for these cassettes that can go in the `part_id`?

Reply:
I have not found pictures of any cassettes for the MSX version so I did not set the part_ids.